docs(moderator): create onboarding guide for fCC moderators (#47938)
freeCodeCamp/freeCodeCamp: docs/moderator-onboarding-guide.md

# The Official freeCodeCamp Moderator Onboarding Guide

This guide will help new moderators get up-and-running with the processes and procedures followed by experienced moderators on the fCC community forum and other official communities that we foster.

> [!NOTE]
> If you haven't read [The Moderator Handbook](https://contribute.freecodecamp.org/#/moderator-handbook) yet, you should start there first.

## The Forum

### First Steps

> The first thing you may notice after being given moderator status on the forum is that your interface will look somewhat different with new admin tools to explore and access to the Mod-Team subforum as well.

Reviewer: ```suggestion
The first thing you may notice after being given moderator status on the forum is that your interface will look somewhat different, with new admin tools to explore and access to the Mod-Team subforum.
```

Grammar and repetition nitpicks

> Some of the new tools will appear inside a new menu item that looks like a wrench. Some will appear as new tabs or buttons or even new enabled options within the forum menus.

Reviewer: ```suggestion
Some of the new tools will appear inside a new menu item that looks like a wrench. Some will appear as new tabs or buttons, or even new enabled options within the forum menus.
```

> To get familiar with the new tools and powers you can combine one or more of the following methods during your first week with this elevated role. (The first two of which are the most important.)

Reviewer: ```suggestion
To get familiar with the new tools and powers, you can combine one or more of the following methods during your first week with this elevated role:

> [!TIP]
> The first two are the most important.
```

### Become Familiar with the Discourse Admin Tools

> The freeCodeCamp forum is actually a Discourse forum and follows many of the same guidelines of other forums built on Discourse. To begin to get familiar with Discourse and the moderation role, start by reading Discourse's [new user guide](https://meta.discourse.org/t/discourse-new-user-guide/96331) then read Discourse's [new moderator guide](https://meta.discourse.org/t/discourse-moderation-guide/63116).

Reviewer: ```suggestion
The freeCodeCamp forum is actually a Discourse forum and follows many of the same guidelines of other forums built on Discourse. To begin to get familiar with Discourse and the moderation role, start by reading Discourse's [new user guide](https://meta.discourse.org/t/discourse-new-user-guide/96331) and Discourse's [new moderator guide](https://meta.discourse.org/t/discourse-moderation-guide/63116).
```

### Shadow a Mod

> All moderator actions can be followed by reviewing the [action logs](https://forum.freecodecamp.org/admin/logs/staff_action_logs). The actions taken by automated tools like 'akistmet' or 'system' can mostly be ignored until they result in a post that needs to be reviewed. Posts to be reviewed will show up in the [Review](https://forum.freecodecamp.org/review) area of the forum.

Reviewer: ```suggestion
All moderator actions can be followed by reviewing the [action logs](https://forum.freecodecamp.org/admin/logs/staff_action_logs). The actions taken by automated tools like `akistmet` or `system` can mostly be ignored until they result in a post that needs to be reviewed. Posts to be reviewed will show up in the [Review](https://forum.freecodecamp.org/review) area of the forum.
```

Code tags for i18n.

> For the first week or so you will want to pay attention to what is getting flagged and what is being reviewed and compare that to the actions being taken upon the flagged posts. So you may see the system account flag a post because the user created it too quickly. In many cases, the moderators will unflag the post by clicking on the "Approve Post" button or mark it as "Not Spam" (depending on the flag type).

Reviewer: ```suggestion
For the first week or so you will want to pay attention to what is getting flagged and what is being reviewed, and compare that to the actions being taken upon the flagged posts. You may see the system account flag a post because the user created it too quickly. In many cases, the moderators will unflag the post by clicking on the "Approve Post" button or mark it as "Not Spam" (depending on the flag type).
```

> Commonly, spam flags can also be raised by members or moderators. Common duplicitious behaviour would involve opening an account, making a post that seems harmless, then editing that post later on to add a link to an external site for the purpose of advertising it. In this case, usually the member account is fairly new and has only made this one post thus far, which indicates that the account was opened for the sole purpose of spamming the community. The decision should be made to delete the account after the first offence in this case. The same applies for accounts whose first post is deemed to be spam.

Reviewer: ```suggestion
Commonly, spam flags can also be raised by members or moderators. Common duplicitous behaviour would involve opening an account, making a post that seems harmless, then editing that post later on to add a link to an external site for the purpose of advertising it. In this case, usually the member account is fairly new and has only made this one post thus far, which indicates that the account was opened for the sole purpose of spamming the community. The decision should be made to delete the account after the first offence in this case. The same applies for accounts whose first post is deemed to be spam.
```

sp.

> You may notice moderators performing a procedure called 'split topic'. Which can be interpreted as either a moderator has split a post that was made in error into the tail-end of a topic that is aleady closed into its own thread for eg.; or a moderator merged duplicate topics that a single user has created for the same fCC challenge. Watching this procedure will highlight different actions and their causes.

Reviewer: ```suggestion
You may notice moderators performing a procedure called 'split topic'. This may be a case where a moderator has split a post that was made erroneously on an existing topic into a new topic, or a moderator merged duplicate topics that a single user has created for the same question. Watching this procedure will highlight different actions and their causes.
```

This one was tricky to read, IMHO.

Author: yes I believe it! It's almost a reflection of my own turmoil!

> Another useful feature that becomes enabled for all moderators is the ability to post a "Canned Reply" which is a pre-formed response that was worked out with the Mod team in order to quickly respond to some well-known and repetitive scenarios. These include:

Reviewer: ```suggestion
Another useful feature that becomes enabled for all moderators is the ability to post a "Canned Reply" which is a pre-written response that was worked out with the mod team in order to quickly respond to some well-known and repetitive scenarios. These include:
```

- welcoming a new forum member who has posted code without a question -> "Welcome - remind question"
- reminding members not to post code solutions but to provide hints and tips instead -> "Solutions Instead of Help"
- responding to a situation where someone's code works for you but not for them -> "Browser Issues"
- encouraging members to open github issues when a possible bug is found -> "Bug Report"

> There are quite a few more which should either be self-explanatory or if not, they should be explained in a post in the Mod team subforum.

Reviewer: I'd remove this line. Calling something "self-explanatory" can discourage someone when they *don't* understand it.

Author: How about:

There are quite a few more which you can read through to become familiar with their respective uses. Some of the history describing how these canned responses came to be or are in use may be found in the posts on the Mod-team subforum.

Reviewer: ```suggestion
There are more, which you can read through to become familiar with their respective uses. You can also find discussion around the templates in the mod-chat subforum, and you are welcome to ask questions if you aren't sure how a template should be used.
```

How's this?

Author: Pretty good but I would want to use the Mod-team name as that is the actual (somewhat unfortunate) name of the subforum.

There are many more that you can read through to become familiar with their respective uses. You can also find prior discussions around the templates in the Mod-chat subforum where you are welcome to ask further questions as needed.

Reviewer: Whoops I messed up the name lol. Edited the suggestion.
